apps/protocol-frontend/src/utils/requireAuth.tsx

```diff
@@ -0,0 +1,61 @@
+import { Box, Container, Stack, Text } from '@chakra-ui/react';
+import ConnectWallet from '../components/ConnectWallet';
+import React from 'react';
+import { useAccount } from 'wagmi';
+import { useAuth } from '../contexts/AuthContext';
+import { GovrnCta } from '@govrn/protocol-ui';
+
+type RequireAuthProps = {
+  children: React.ReactNode;
+};
+
+const NonAuthenticatedView = () => {
+  return (
+    <GovrnCta
+      heading={'Welcome to Govrn'}
+      emoji={'🫡'}
```

Comment:
I'm thinking that the particular emoji we used in the Figma may not be fully supported.

We can replace it with 👋 - Should be able to copy this directly from the one in the Dashboard.tsx